packages/@ourworldindata/core-table/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "@ourworldindata/core-table",
+  "version": "0.0.0",
+  "description": "This is our core tabular data structure class which parses JSON or delimited data and makes it available for charts. This class performs core transformations like filtering, grouping, etc. This is roughly our \"Pandas for TypeScript\".",
+  "main": "dist/index.js",
+  "directories": {
+    "lib": "src"
+  },
+  "types": "dist",
+  "scripts": {
+    "build": "tsc",
+    "watch": "tsc --watch"
+  },
+  "keywords": [],
+  "author": "",
+  "dependencies": {
+    "@ourworldindata/utils": "*",
+    "@types/d3": "^6",
+    "d3": "^6.1.1",
+    "dayjs": "^1.11.5",
+    "eslint": "^8.23.1",
+    "fast-cartesian": "^5.1.0",
+    "prettier": "^2.7.0",
+    "typescript": "~4.8.2"
```

Comment:
also a nit, and we don't need to address this now: but we can make our split between `dependencies` and `devDependencies` more sensible now (and should do this before we publish to npm), and typescript is surely a `devDependency`.